plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -431,4 +433,30 @@ public DeltaLakeConfig setUniqueTableLocation(boolean uniqueTableLocation)
         this.uniqueTableLocation = uniqueTableLocation;
         return this;
     }
+
+    public boolean isCreateTableWithExistingLocationEnabled()
+    {
+        return createTableWithExistingLocationEnabled;
+    }
+
+    @Config("delta.create-table-with-existing-location.enabled")
```

Comment:
We include `legacy` or `deprecated` keyword when adding a deprecated config property.